bot.py

```diff
@@ -25,26 +34,39 @@ async def format_embed(self, name, anime=False):
         except kadal.MediaNotFound:
             return
 
-        title = (media.title.get('english')
-                 or media.title.get('romaji')
-                 or media.title.get('native'))
-        desc = "***" + ", ".join(media.genres) + "***\n"
+        title = (media.title.get("english")
+                 or media.title.get("romaji")
+                 or media.title.get("native"))
+        desc = "**" + ", ".join(media.genres) + "**\n"
         if media.description is not None:
             desc += media.description[:256 - len(desc)] + f"... [(more)]({media.site_url})"
         # dirty half-fix until i figure something better out
-        desc = desc.replace("<br>", "").replace("<i>", "").replace("</i>", "")
-        footer = re.sub(r'.*\.', '', str(media.format))
-        e = discord.Embed(title=title, description=desc, color=0x4286f4)
-        e.set_footer(text=footer.replace("TV", "ANIME"))
-        e.set_thumbnail(url=media.cover_image)
+        desc = re.sub(r"<[bri/]{1,2}>", "", desc, flags=re.I|re.M)
+        footer = re.sub(r".*\.", "", str(media.format))
+        
+        async with aiohttp.ClientSession() as session:
+            async with session.get(f"{self.anilist_cover_url}{media.id}") as resp:
+                if resp.status == 200:
```

Comment:
You need to handle the case where its not 200, this will throw, since the color is defined inside but used outside. (the image in set_image would also be invalid, but im pretty sure discord handles that on itself so its up to you on that front)

Reply:
If the image doesn't load I'm pretty sure it just doesn't load at all, but we could maybe do some tests to make sure it wouldn't become an issue